src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractKeyspaceIntegrationTestBase.java

```diff
@@ -50,6 +60,25 @@ public void createKeyspace() {
     createKeyspace(keyspaceName);
   }
 
+  @BeforeAll
+  public void createDriverSession() {
```

Comment:
Do we want to always create session for every test? Maybe it could be created on demand instead?

Also: should this be closed after tests are run so as not to leave connections hanging?